src/main/webapp/app/admin/user-management/user-management.route.ts

```diff
@@ -23,53 +23,66 @@ export class UserMgmtResolve implements Resolve<any> {
     }
 }
 
-export const userMgmtRoute: Route = {
-    path: 'user-management',
-    component: UserManagementComponent,
-    resolve: {
-        pagingParams: JhiResolvePagingParams,
-    },
-    data: {
-        pageTitle: 'userManagement.home.title',
-        defaultSort: 'id,asc',
+export const userMgmtRoute: Route[] = [
+    {
+        path: 'user-management',
+        component: UserManagementComponent,
+        resolve: {
+            pagingParams: JhiResolvePagingParams,
+        },
+        data: {
+            pageTitle: 'userManagement.home.title',
+            defaultSort: 'id,asc',
+        },
     },
-    children: [
-        {
-            path: 'new',
-            component: UserManagementUpdateComponent,
-            resolve: {
-                user: UserMgmtResolve,
-            },
-            data: {
-                pageTitle: 'userManagement.home.createLabel',
-            },
+    {
+        // Create a new path without a component defined to prevent resolver the UserManagementComponent from being always rendered
```

Comment:
```suggestion
        // Create a new path without a component defined to prevent resolver caching and the UserManagementComponent from being always rendered
```